### DISCUSSION TITLE
高DPI显示问题

### DISCUSSION THREAD

**Discussion by xjjww** [tamlok/vnote#2106] [Q&A]
一开始vnote在我的电脑上显示正常，有一次闪退后重新打开，vnote的各种字体变得很大，像是放大了150%（正好是我的系统设置）。该怎么做才能恢复原状？

**Comment by tamlok** [comment]
EXE上面右键菜单DPI什么的，搜一下windows怎么设置吧

**Reply by xjjww** [reply]
已解决，多谢
